frontend/src/components/misc/GuideWrapper.jsx

```diff
@@ -0,0 +1,272 @@
+import React from "react";
+import Joyride from "react-joyride";
+import { Outlet, useNavigate } from "react-router-dom";
+import { useMount } from "react-use";
+import { useGuideContext } from "../../contexts/GuideContext";
+
+export default function GuideWrapper() {
+  const { guideState, setGuideState } = useGuideContext();
+  const navigate = useNavigate();
+
+  const steps = [
+    {
+      target: "#home__bgImg",
+      content: (
+        <div id="guidebox">
+          <h3>Guide</h3>
+          <p>
+            Welcome to IntelOwls Guide for First Time Visitors!, for further
+            questions you could check out our <a href="/">docs</a> or reach us
+            out on <a href="/">intelowls slack channel</a>
+          </p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#Dashboard_title",
+      content: (
+        <div id="guidebox">
+          <h3>Dashboard</h3>
+          <p>See previous job details here with charts and more</p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
+      target: "#Dashboard_timepicker",
+      content: (
+        <div id="guidebox">
+          <h3>Filter</h3>
+          <p>Filter by time to get details about previous jobs</p>
+        </div>
+      ),
+      disableBeacon: true,
+    },
+    {
```

Comment:
on a second thought, I think that the dashboard section could be the last one to be viewed. This is because I think that first we have to explain how IntelOwl works